corehq/apps/hqcase/api/get_list.py

```diff
@@ -93,7 +92,16 @@ def get_list(domain, params):
         else:
             raise UserError(f"'{key}' is not a valid parameter.")
 
-    return [serialize_es_case(case) for case in query.run().hits]
+    es_result = query.run()
+    hits = es_result.hits
+    ret = {
+        "total": es_result.total,
+        "cases": [serialize_es_case(case) for case in hits],
+    }
+    cases_in_result = len(hits)
+    if cases_in_result and es_result.total > cases_in_result:
+        ret['next'] = {**raw_params, **{'indexed_on.gt': hits[-1]["@indexed_on"]}}
```

Comment:
@snopoke I think we spoke about this but can you comment on my choice to use `gt` instead of `gte` here?  `gt` is nice because pages would be non-overlapping, but if there are any documents inserted at the same microsecond, there's a chance of incomplete results.  I think I need to change this unless we're certain these values are unique.

Reply:
I don't think we can guarantee uniqueness on the dates so going with `gte` is likely necessary. I don't know who much of an issue it is to have overlap in the results - we could deduplicate that in the view though it would be a pain.

Reply:
Yeah, gonna do that for now.  I imagine deduplication would look like this:
* Add secondary ordering by UUID
* Pass the last seen doc's UUID as a param in `next`
* Fetch all docs with that `@indexed_on` value, keep any after the UUID, discard the rest
* Do another query with an appropriate page size to finish the request.

Assuming duplicates are at least rare, I guess we could also just fetch `page_size + 5` or something and process it in python to return the proper number of results.

It'd be nice to avoid that, but I could be persuaded.